tangelo/linq/helpers/circuits/clifford_circuits.py

```diff
@@ -0,0 +1,71 @@
+from math import pi, isclose
+
+from tangelo.linq import Gate
+
+
+def decompose_gate_to_cliffords(gate, abs_tol=1e-4):
```

Comment:
Open question: this could also be defined in `gate.py`, either as a free function or a method of the `Gate` class. It purely manipulates abstract gates without need for the definition of `Circuit`.

But it may be useful to have an explicit module about Clifford circuits helpers !